crates/oxc_linter/src/config/config_builder.rs

```diff
@@ -1,12 +1,12 @@
+use itertools::Itertools;
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_span::CompactStr;
+use rustc_hash::FxHashSet;
```

Comment:
Is this something wrong with my setup that moves the use statements? I've noticed it quite a bit.

I use RustRover, so it could be that I've got it setup in some way that conflicts with the existing code. If that's the case, let me know and I can review my configuration.

Reply:
You probably have some kind of auto import sorting plugin turned on. But yes, please change the import order to furtherest to closest.